ramls/types/identifier.json

```diff
@@ -14,13 +14,13 @@
     "subtype": {
       "type": "string",
       "description": "Subtype for the identifier. Valid values are  Print, Online.",
-      "enum": [ "Print", "Online"],
+      "enum": [ "Empty", "Print", "Online", "Preceding", "Succeeding", "Regional", "Linking", "Invalid"],
```

Comment:
Do we need the other types if we are only concerned with Print and Online and ISSN and ISBN?

Reply:
I felt like we should not need them but when I copied a payload from UI and used that for PUT, then our code was blowing up. But, now that I think about it, the reason the payload in the UI contained other types is because Ruby module is not filtering those in the GET response for resourceId where as we filter it in Java module now. So, probably, there should not be a situation where a PUT from UI using Java code should supply other types other than Print/Online. I will remove these and hopefully it should be fine.